builder.py

```diff
@@ -130,6 +131,15 @@ def main():
 			# Enlist file paths to be zipped
 			files_to_zip = retrieve_file_paths(dir_top_level)
 			
+			# Create directory to store the resulting zip files in
+			dir_path = "./Problems"
+
+			if not os.path.exists(dir_path):
+				print( "Creating folder")
```

Comment:
Take this print statement after the folder is created and change the payload to:

`print("Created Output Directory")`

Reply:
Print statement changed in the if condition.